anubis/entrypoints/puppet-strings

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+# Establish source dir for relative includes
+DIR="${BASH_SOURCE%/*}"
+if [[ ! -d "$DIR" ]]; then DIR="$PWD"; fi
+
+# Pre-suite (download and extract requested release)
+# shellcheck source=../shared/entrypoint-pre disable=SC1091
+. "$DIR/../shared/entrypoint-pre"
+
+# Make sure required Anubis env vars are set
+# shellcheck source=../shared/entrypoint-anubis-pre disable=SC1091
+. "$DIR/../shared/entrypoint-anubis-pre"
+
+# TODO make a PDK frontend/backdoor for this? like a wrapper in /opt/puppetlabs/pdk/private?
+export PATH=$PATH:/opt/puppetlabs/pdk/private/ruby/2.5.7/bin
+export GEM_PATH=/opt/puppetlabs/pdk/private/ruby/2.5.7/lib/ruby/gems/2.5.0:/opt/puppetlabs/pdk/share/cache/ruby/2.5.0:/opt/puppetlabs/pdk/private/puppet/ruby/2.5.0
+
+# Run strings and emit output to json document
+/opt/puppetlabs/pdk/private/puppet/ruby/2.5.0/bin/puppet strings generate --format json | tee anubis_output.json
```

Comment:
If we want to use PDK for all of the Anubis validations, being able to run `pdk validate` commands without the module having been `pdk convert`-ed will become a requirement. Otherwise we'll have to resort to things like this to bypass `bundle install` steps.